airflow/www/templates/airflow/trigger.html

```diff
@@ -27,11 +27,93 @@
+          {%- if not "null" in form_details.schema.type %} required=""{%- endif %}
+          value="{% if form_details.value %}{{ form_details.value[0:2] }}00-01-01 {{ form_details.value }}{% endif %}" />
+      </div>
+    {% elif "enum" in form_details.schema and form_details.schema.enum %}
+      <select class="my_select2 form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" data-placeholder="Select Value"
+        value="{% if form_details.value %}{{ form_details.value }}{% endif %}" onchange="updateJSONconf();"
+        {%- if not "null" in form_details.schema.type %} required=""{% endif %}>
+        {% for option in form_details.schema.enum -%}
+        <option>{{ option }}</option>
+        {% endfor -%}
+      </select>
+    {% elif form_details.schema and "array" in form_details.schema.type %}
+      <textarea class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" valuetype="array" rows="6"
+        {%- if not "null" in form_details.schema.type %} required=""{% endif -%}>
+        {%- for txt in form_details.value -%}
+          {{ txt }}{{ "\n" }}
+        {%- endfor -%}
+      </textarea>
+    {% elif form_details.schema and "object" in form_details.schema.type %}
+      <textarea class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" valuetype="object" rows="6"
+        {%- if not "null" in form_details.schema.type %} required=""{% endif -%}>
+        {{- form_details.value | tojson() -}}
+      </textarea>
+    {% elif form_details.schema and ("integer" in form_details.schema.type or "number" in form_details.schema.type) %}
+      <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" placeholder="" valuetype="number" type="number"
+        value="{% if form_details.value %}{{ form_details.value }}{% endif %}"
+        {%- if form_details.schema.minimum %} min="{{ form_details.schema.minimum }}"{% endif %}
+        {%- if form_details.schema.maximum %} max="{{ form_details.schema.maximum }}"{% endif %}
+        {%- if form_details.schema.type and not "null" in form_details.schema.type %} required=""{% endif %} />
+    {% else %}
+      <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" placeholder="" type="text"
+        value="{% if form_details.value %}{{ form_details.value }}{% endif %}"
+        {%- if form_details.schema and form_details.schema.minLength %} minlength="{{ form_details.schema.minLength }}"{% endif %}
+        {%- if form_details.schema and form_details.schema.maxLength %} maxlength="{{ form_details.schema.maxLength }}"{% endif %}
+        {%- if form_details.schema and form_details.schema.type and not "null" in form_details.schema.type %} required=""{% endif %} />
+    {% endif %}
+    {% if form_details.description -%}
+      <span class="help-block">{{ form_details.description }}</span>
```

Comment:
This makes the help text/description either as markup (if it is HTML) or plain text:
![image](https://user-images.githubusercontent.com/95105677/214951601-2efa51b2-5787-4351-851f-806706956767.png)